homeassistant/components/august/__init__.py

```diff
@@ -142,10 +148,42 @@ async def async_setup(self):
         self._remove_inoperative_locks()
         self._remove_inoperative_doorbells()
 
+        pubnub = AugustPubNub()
+        for device in self._device_detail_by_id.values():
+            pubnub.register_device(device)
+
         self.activity_stream = ActivityStream(
-            self._hass, self._api, self._august_gateway, self._house_ids
+            self._hass, self._api, self._august_gateway, self._house_ids, pubnub
         )
         await self.activity_stream.async_setup()
+        pubnub.subscribe(self.async_pubnub_message)
+        self._pubnub_unsub = async_create_pubnub(user_data["UserID"], pubnub)
+
+    @callback
+    def async_pubnub_message(self, device_id, date_time, message):
```

Comment:
I can move part of this into the pypi since I'm forking it now and reduce the code here